clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerResourceDefinition.java

```diff
@@ -207,6 +209,16 @@ public ModelNode transformOperation(ModelNode operation) {
                 }
             };
             builder.addRawOperationTransformationOverride(ListOperations.LIST_REMOVE_DEFINITION.getName(), new SimpleOperationTransformer(removeAliasTransformer));
+
+            // Reject if using embedded thread pool configuration
+            builder.rejectChildResource(ThreadPoolResourceDefinition.WILDCARD_PATH);
+        } else {
+            for (ThreadPoolResourceDefinition pool : ThreadPoolResourceDefinition.values()) {
+                pool.buildTransformation(version, parent);
+            }
+            for (ScheduledThreadPoolResourceDefinition pool : ScheduledThreadPoolResourceDefinition.values()) {
+                pool.buildTransformation(version, parent);
+            }
```

Comment:
Yes, that's right Paul, this is a problem caused by rebasing, since this was originally developed for WildFly 9 and thus schema 3_0. I fixed that when rebasing on most places but this one got through...

Sent from my iPhone

> On 07 Aug 2015, at 22:43, Paul Ferraro <notifications@github.com> wrote:
> 
> In clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerResourceDefinition.java:
> 
> > @@ -207,6 +209,16 @@ public ModelNode transformOperation(ModelNode operation) {
> >                  }
> >              };
> >              builder.addRawOperationTransformationOverride(ListOperations.LIST_REMOVE_DEFINITION.getName(), new SimpleOperationTransformer(removeAliasTransformer));
> > +
> > +            // Reject if using embedded thread pool configuration
> > +            builder.rejectChildResource(ThreadPoolResourceDefinition.WILDCARD_PATH);
> > +        } else {
> > +            for (ThreadPoolResourceDefinition pool : ThreadPoolResourceDefinition.values()) {
> > +                pool.buildTransformation(version, parent);
> > +            }
> > +            for (ScheduledThreadPoolResourceDefinition pool : ScheduledThreadPoolResourceDefinition.values()) {
> > +                pool.buildTransformation(version, parent);
> > +            }
> I think this might be the reason why this PR broke support for legacy slaves. Shouldn't this transformation logic be triggered based on InfinispanModel.VERSION_4_0 instead of InfinispanModel.VERSION_3_0?
> 
> —
> Reply to this email directly or view it on GitHub.
>